src/bci_build/package/__init__.py

```diff
@@ -527,6 +527,10 @@ class BaseContainerImage(abc.ABC):
     #: present
     logo_url: str = ""
 
+    #: Optional custom build name that will override the default buildname, if
+    #: set
+    custom_build_name: str = ""
```

Comment:
I'm just afraid that it will run out of sync, and then it will be confusing or bite us. I think a boolean would be better.

Reply:
implemented the boolean variant in #1124